src/api-endpoints.ts

```diff
@@ -10192,7 +10207,10 @@ type AppendBlockChildrenPathParameters = {
   block_id: IdRequest
 }
 
-type AppendBlockChildrenBodyParameters = { children: Array<BlockObjectRequest> }
+type AppendBlockChildrenBodyParameters = {
+  children: Array<BlockObjectRequest>
+  after?: IdRequest
```

Comment:
@rhart92, is there a description of what the `after` parameter is used for? I don't see it mentioned in the [documentation](https://developers.notion.com/reference/patch-block-children), but it sounds intriguing!

Reply:
Hey @dvanoni. The documentation is coming 🔜 but for now if you want to append blocks to somewhere other than then end of the parent block, you can try passing the block ID you want to append after in the `after` prop ⚡ Essentially it just provides a bit more customizability to the [Append block children](https://developers.notion.com/reference/patch-block-children) endpoint.

Reply:
Thanks for the info! I was hoping that's what it did. Sounds super useful!